docs/excel/custom-functions-architecture.md

```diff
@@ -0,0 +1,50 @@
+---
+ms.date: 03/20/2019
+description: Learn about Excel custom functions' runtime. 
+title: Custom functions architecture (preview)
+localization_priority: Priority
+---
+# Custom functions architecture
+
+ Custom functions are with their own unique runtime that prioritizes execution of calculations. This article will cover the differences between the custom functions runtime and the browser-based JavaScript engine which powers most other parts of your add-in.
+
+## Custom functions runtime
+
+Because of their unique runtime, think of custom functions as another component of your total add-in. Your add-in will also likely include elements from the browser engine runtime, like a task pane.
```

Comment:
Maybe we should just list all the parts to make this clearer. Would make a good picture actually to show visually how it all runs (perhaps later).

"An Office Web Add-in can interact with the user as a task pane, or a content pane, and can include commands and custom functions. All of these parts run in a browser engine runtime except for custom functions. Custom functions run in a separate custom functions runtime to optimize for calculation speed..." 

or something like that.